extra/1-currency-conversion.js

```diff
@@ -15,7 +17,10 @@ function convertToUSD() {}
   They have also decided that they should add a 1% fee to all foreign transactions, which means you only convert 99% of the £ to BRL.
 */
 
-function convertToBRL() {}
+function convertToBRL(price) {
+  const BRL = (price * 0.99 * 5.7).toFixed(2);
```

Comment:
please note, 
Number.toFixed() method might behave dodgy 
here are the discussions: 
- https://github.com/denysdovhan/wtfjs#numbertofixed-display-different-numbers
- https://stackoverflow.com/q/43803587
that's related to different browsers.

to avoid that, you can use alternative way of rounding: 

```js
Math.round((a * b) * 100) / 100;
```